templates/listing_show.html

```diff
@@ -0,0 +1,23 @@
+<!-- Make into macro -->
```

Comment:
Flask might have its own comment syntax (for things that are meant to be server-only).